seemabansal2003.md

```diff
@@ -0,0 +1,55 @@
+---
+name: "@seemabansal2003"
+project: "TechnoFlora"
+---
+
+# TechnoFlora
+
+## Summary
+
+I come from a family of gardening enthusiasts who love to spend time with their plants. However whenever we go on a vacation, most of our plants die due to lack of care. Thus, being an Arduino Enthusiast, i want to build an automatic hydroponics based plant incubator called TechnoFlora. TechnoFlora would feature technologies like hydroponics, grow lights & temperature and humidity control that would not only help the plant in growing during adverse conditions but also faster than normal during better times. I can't wait to grow and completely harvest lettuce within just a couple of weeks!
+
+## Plan
+
+First of all, TechnoFlora would feature a hydroponics tray with water level indicators that can act as the medium of plant growth, nourishing the plant with the required minerals. Then it would also feature a full spectrum grow light which can artificially stimulate photosynthesis and help plants grow faster. Along with this, It would be covered with an acrylic enclosure that would not only trap stray light but also help in making a controlled environnment around the plant. There would be a temperature & humidity control system featuring a DHT11 sensor & vntilation and cooling fans to control the plants surroundings. All information regarding the plant's growth would be synced with a mobile application using bluetooth. This would not only help me monitor the plant better but also help it grow faster.
+
+## Budget
+
+
+| Product                                                | Supplier/Link                     | Cost   |
+| --------------------------------------------------------------------- | ---------------------------------------------- | ---------- |
+|hydroponics tray |https://amzn.eu/d/4M5MgpP| $43.50 |
```

Comment:
what's the reason for purchasing this specific hydroponics tray? if you're making your own plant incubator wouldn't a lot of the features here be redundant?

Reply:
This hydroponics tray already comes with Industrial grade yellow grow lights  (used only for leaf growth)  & on integrating them with the other grow lights that i have mentioned, we can create a full spectrum grow light that is suitable for all growth requirements. Plus, it provides a nice & steady base, supports 9-10 plants at a time (which is quite a lot for hydroponic planters) & also has features like water level indicators which greatly reduces my work. Also it already comes with other needed supplies like a 12v 2A Power Adapter,Algae proofing caps & Foam Grow cubes that would otherwise need to be bought separately. That's why i feel like it's much better to modify an existing setup rather than start from scratch.

Reply:
sounds good, thank you!